README.md

```diff
@@ -52,6 +52,31 @@ in napalm.buildPackage ./. { packageLock = <path/to/package-lock>; }
 
 If you want to use Napalm in your flake project, you can do that by adding it to your inputs and passing `napalm.overlay` to your Nixpkgs instance. There is also a template, that can help you use napalm in your project. You can use it by running `nix flake init -t "github:nix-community/napalm"`.
 
+**Example**
```

Comment:
Also maybe use a level four heading instead of a fake one? That should work better for screen readers.

Reply:
Did that since thats how all the other examples were indicated. I can fix this one, but would you like me to make this change to all of the others as well?

Reply:
Oh, I missed that. If you could change the others, that would be great.

Reply:
Fixed this one and all others in 9e76864